src/config.rs

```diff
@@ -353,6 +373,45 @@ pub fn empty_to_none<A: AsRef<str>>(inp: Option<A>) -> Option<A> {
     inp
 }
 
+pub fn find_rootca_for_xds(citadel_ca_cert_path: String) -> Result<RootCert, Error> {
```

Comment:
The existing istio implementation is legacy - we have to handle all sorts of old junk. I don't think we need most of this at all today. Probably just default to `./var/run/secrets/istio/root-cert.pem` and allow a single env to override.

BTW: ca_root_cert should have similar logic (but separate config)

Reply:
Hi @howardjohn, okay, There should be 2 envs for xds and ca root cert, such as `XDS_CERT_PROVIDER` for xds root cert and  `CA_CERT_PROVIDER` for ca root cert. And both of them have `./var/run/secrets/istio/root-cert.pem` by default.